### ISSUE TITLE
Migrate documentation from GitHub Wiki to docs.microsoft.com

### ISSUE THREAD

**Issue by brunoborges** [microsoftgraph/msgraph-sdk-java#122]
The documentation of this library must reside under docs.microsoft.com.

Please consider this for the next planning.

**Comment by darrelmiller** [comment]
Hey @brunoborges!  I'm the PM for the SDK projects.  I wasn't aware of this requirement.  Can you point me to where I can find out more about it and where we can find resources to make this possible?

**Comment by brunoborges** [comment]
Hey @darrelmiller, ping me at `brborges@ms...`

**Comment by baywet** [comment]
@darrelmiller : I know this conversation was a long time ago, can you let me know if there is still any work to be done here? otherwise can we close the issue? Thanks!

**Comment by darrelmiller** [comment]
Yes.  the docs platform can now generate reference documentation for Java.  We should generate reference documentation for the core library like we do for .NET.  Also, we should continue to look for docs that are missing Java samples.  Many of our reference pages do, but some of the conceptual pages are missing the Java Samples.

**Comment by baywet** [comment]
Thanks. Sure, please start providing any resource you think useful for that here. And I'll take a stab at it when I get to it :)

**Comment by baywet** [comment]
https://review.docs.microsoft.com/en-us/help/onboard/java-requirements?branch=master
